pysat/instruments/methods/nasa_cdaweb.py

```diff
@@ -273,7 +273,7 @@ def download(supported_tags, date_array, tag, sat_id,
         if not multi_file_day:
             try:
                 logger.info(' '.join(('Attempting to download file for',
-                                date.strftime('%d %B %Y'))))
+                            date.strftime('%d %B %Y'))))
```

Comment:
I don't think some of PEP8 formatting will ever look 'right' to me. So it goes. I'd actually push the lines more to the right to emphasize they are in the inner parenthesis.

Reply:
D'oh.  You're right.  These should be to the right of `join`.

Reply:
fixed.